client/lib/transformer.dart

```diff
@@ -154,3 +162,5 @@ const _ANALYTICS_CODE = '''<script>
   ga('send', 'pageview');
 
 </script>''';
+
+const _SURVEY_CODE = '<script async="" defer="" src="//survey.g.doubleclick.net/async_survey?site=4vq7qmqpfn4zjduve64c4h6lqa"></script>';
```

Comment:
So we should probably be doing the same thing as with the analytics - it looks for a file with the information that isn't in the dartdoc-viewer, but is present in the deployment setup.